### PR TITLE
refactor: done building services layer 🔃

### PR DESCRIPTION
**Pull request by yusufmsabeh**

### CODE REVIEW

File: src/services/category.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import {Category} from "../models/Category";
+
+export const getCategories = async (options={}):Promise<Category[]>=>{
```
Comment (mshareef-kiitos) [inline src/services/category.ts]: You can add the type of the options: `(options:  FindOptions = {})`. You can get `FindOptions` type from sequelize package.

File: src/controllers/orders.ts [hunk 2]
```diff
@@ -29,19 +28,15 @@ export const postOrders: RequestHandler = async (
         transaction_id: transactionId,
       });
     } else if (email) {
-      order = await Order.create({
+      order = await orderServices.createOrder({
         email: email,
         status: ORDER_STATUS.ACTIVE,
```
Comment (mshareef-kiitos) [inline src/controllers/orders.ts:34]: Where `ORDER_STATUS` has been imported in this page?

File: src/services/order.ts [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import {Order} from "../models/Order";
+
+export const createOrder = async (order:OrderAttributes)=>{
+  return await Order.create({ ...order });
+};
+
+
+interface OrderAttributes {
+    email: string,
+    status: ORDER_STATUS,
```
Comment (mshareef-kiitos) [inline src/services/order.ts:11]: Where `ORDER_STATUS` has been imported in this page?

**Review by mshareef-kiitos — CHANGES_REQUESTED** [review]

File: src/controllers/orders.ts (hunk 2 above)
Comment (yusufmsabeh) [reply src/controllers/orders.ts:34]: I added imports statement 👍

File: src/services/order.ts (hunk 3 above)
Comment (yusufmsabeh) [reply src/services/order.ts:11]: I added imports statement 👍

File: src/services/category.ts (hunk 1 above)
Comment (yusufmsabeh) [reply src/services/category.ts]: Yes, that's a good idea, I used FindOptions type

**Pull request merged**